packages/server/modules/blobstorage/services/index.js

```diff
@@ -0,0 +1,111 @@
+const knex = require('@/db/knex')
+const {
+  SpeckleNotFoundError,
+  SpeckleResourceMismatch
+} = require('@/modules/shared/errors')
+const BlobStorage = () => knex('blob_storage')
+
```

Comment:
i don't like the approach of creating a service called `index.js` in our modules - what if we want to create other services? also, I think such an approach encourages devs to grow the file to be bigger and handle more concerns than it needs to

Reply:
makes sense, but its an easy refactor when the stuff gets bigger
and while the blobstorage has only 1 type of service, i was having a hard time comming up with a name for a service that is blobstorage / blobstorage.

BTW what is ur preferred limit of stuff in a single module / file?

Reply:
> BTW what is ur preferred limit of stuff in a single module / file?
There's not a hard line, it's more of a feeling, but i'd say that when you go over 500 LOC you're probably starting to put too much stuff in there. it would be nice if we could even implement this as a linting rule, there's a rule for that.

> its an easy refactor when the stuff gets bigger
agreed, and yea until you actually figure out what those extra concerns are, it's not realistic to just split it apart ahead of time. so I agree that it can remain all in one file for now, but I def disagree with it being called 'index.js' then, cause that kinda marks it as the "main" service, which won't make sense when there will be many. even something like "blobStorageService.js" or "blobManagementService.js" would be better.